configs/config.yaml

```
@@ -0,0 +1,42 @@
+train:
+  batch_size: 64
+  epochs: 100
+  log_interval: 10
+  lr: 1e-6
+
+logger:
+  results_path: "results/"
+  checkpoint_path: "checkpoints/"
+  enable_wandb: False
```

```suggestion
  enable_wandb: True
```

I think it's better to log everything, unless you have a specific reason not to :)

---

Copied from config in the previous repo, wasn't using it. Deleted :)